etc/helm/pachyderm/templates/pachd/config-secret.yaml

```diff
@@ -45,21 +45,28 @@ stringData:
   #   robot:test:
   #   - repoReader
 
+# when registering a cluster with a separate enterprise server
+{{ if .Values.pachd.activateEnterpriseMember }}
+
+  enterpriseServerAddress: {{ required "To register a pachd instance with an enterprise server, the enterprise server's address must be provided." .Values.pachd.enterpriseServerAddress }}
+
+  enterpriseRootToken: {{ required "To register this pachd as an enterprise member, the enterprise root token must be provided." .Values.pachd.enterpriseRootToken }}
+
   # TODO: Add config option for non embedded / multiple pachs?
   # enterpriseClusters is the set of pachds covered by license service 
   enterpriseClusters: |
-    - address: grpc://localhost:1653
```

Comment:
Are we getting rid of the localhost case because it's assumed if it's not passed into the config pod?

Reply:
Yeah it's actually being implied by the top level "enterpriseSecret" entry in the secret. When it's provided, the pachd cluster will set local as a client to itself - aka embedded enterprise

Reply:
So in this case, it wasn't necessary to include the enterpriseClusters / enterpriseConfig declarations until now, since they were being implied by the "enterpriseSecret" config.